src/ctim/schemas/verdict.clj

```diff
@@ -11,7 +16,32 @@ the active verdict.  If there is more than one Judgement with that
 priority, then Clean disposition has priority over all others, then
 Malicious disposition, and so on down to Unknown.
 "
-  {:type (s/enum "verdict")
+  {:type Type
    :disposition c/DispositionNumber
    (s/optional-key :judgement_id) rel/JudgementReference
    (s/optional-key :disposition_name) c/DispositionName})
+
+(s/defschema StoredVerdict
+  "A Verdict as stored in the data store"
+  (st/merge Verdict
+            {:id s/Str
+             :type Type
+             :owner s/Str
+             :created c/Time}))
+
+(s/defn make-id :- s/Str
+  []
+  (str "verdict-" (UUID/randomUUID)))
+
+(s/defn realize-verdict :- StoredVerdict
+  ([new-verdict :- Verdict
+    login :- s/Str]
+   (realize-verdict new-verdict (make-id) login))
+  ([new-verdict :- Verdict
+    id :- s/Str
+    login :- s/Str]
+   (let [now (time/now)]
+     (assoc new-verdict
+            :id id
+            :owner login
+            :created now))))
```

Comment:
The `make-id` and `realised-verdict` should be present only in CTIA not in CTIA. So you could remove them.